### PR TITLE
added hi-IN translations

### PR DESCRIPTION
**Pull request by plutonium-239**
Hey!
I've added hi-IN translations as mentioned in the issue https://github.com/typora/Typora-i18n/issues/283

### CODE REVIEW

File: hi_IN.lproj/Panel.strings [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+
+"Quit" = "छोड़ना";
+"Quit Typora when last window is closed" = "अंतिम विंडो बंद होने पर टाइपोटा को छोड़ दें";
+"On Launch" = "लॉन्च पर";
+"Open new file" = "नई फ़ाइल खोलें";
+"Restore last closed folders" = "अंतिम बंद फ़ोल्डर्स को पुनर्स्थापित करें";
+"Restore last closed files and folders" = "अंतिम बंद फ़ाइलों और फ़ोल्डरों को पुनर्स्थापित करें";
+"Open custom folder" = "खुला कस्टम फ़ोल्डर";
+"Select custom folder" = "कस्टम फ़ोल्डर का चयन करें";
+"This option may be overridden by \"Close windows when quitting an application\"इस विकल्प को ओवरराइड किया जा सकता है \" = "This option may be overridden by \"Close windows when quitting an application\" in System Preferences";
```
Comment (abnerlee) [inline hi_IN.lproj/Panel.strings:240]: @plutonium-239 hi, could you check this line?

File: hi_IN.lproj/Welcome.strings [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+"Unknown Error. Please contact hi@typora.io" = "अज्ञात त्रुटि। कृपया hi@typora.io से संपर्क करें";
+"Failed to access the license server. Please check your network or try again later." = "लाइसेंस सर्वर तक पहुंचने में विफल। कृपया अपना नेटवर्क जांचें या बाद में पुनः प्रयास करें।";
+
+"Enter a license code to support our development" = "हमारे विकास का समर्थन करने के लिए एक लाइसेंस कोड दर्ज करें";
+"Re-enter Email" = "ईमेल पुन: दर्ज करें";
+"Email address confirmation does not match." = "ईमेल पता पुष्टिकरण मेल नहीं खाता है।";
+
+"Are you sure to deactivate Typora on this device?" = "क्या आप इस डिवाइस पर टाइपोरा को निष्क्रिय करना सुनिश्चित कर रहे हैं?";
+
+"Your license has exceeded the max devices numbers.\nIf you click \"Continue Activation\"आपका लाइसेंस अधिकतम डिवाइस संख्याओं को पार कर गया है। \nIf आप क्लिक \" = "Your license has exceeded the max devices numbers.\nIf you click \"Continue Activation\", this device will be activated and the oldest device will be unregistered automatically.";
```
Comment (abnerlee) [inline hi_IN.lproj/Welcome.strings:84]: @plutonium-239 and this line seems also contains errors

File: hi_IN.lproj/Front.strings [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+/* Common */
+"OK" = "ठीक है";
+"Cancel" = "रद्द करें";
+"Yes" = "हाँ";
+"No" = "नहीं";
+"Close" = "बंद करे";
+"Select" = "चुनते हैं";
+"Continue" = "जारी रखें";
+"Learn Moreâ€¦" = "और जानें € |";
```
Comment (abnerlee) [inline hi_IN.lproj/Front.strings:16]: Should I correct to `"Learn More…" = "और जानें…";` ?

File: hi_IN.lproj/Front.strings [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+"Spell Check (%@ unavailable)" = "वर्तनी जाँच (% @ अनुपलब्ध)";
+"Spell Check (%@ downloading)" = "वर्तनी जांच (% @ डाउनलोडिंग)";
+"Download dictionary file for %@" = "% @ के लिए शब्दकोश फ़ाइल डाउनलोड करें";
+"(Default spell check option)" = "(डिफ़ॉल्ट वर्तनी जांच विकल्प)";
+"Failed to load spellcheck module" = "स्पेलचेक मॉड्यूल लोड करने में विफल";
+
+"<strong>%@</strong> has been turned on." = "<strong>% @ </ strong> चालू किया गया है।";
+"You could turn it off from `View` menu." = "आप इसे 'व्यू` मेनू से बंद कर सकते हैं।";
+"The file is too large to render in Typora." = "टाइपोरा में प्रस्तुत करने के लिए फ़ाइल बहुत बड़ी है।";
+"Open inâ€¦" = "€ € | में खुला";
```
Comment (abnerlee) [inline hi_IN.lproj/Front.strings:161]: could you correct translation for `open in…` ?

File: hi_IN.lproj/Panel.strings [hunk 5]
```diff
@@ -0,0 +1,487 @@
...
+"Always Show Word Count" = "हमेशा शब्द गिनती दिखाएं";
+"Font Size" = "फ़ॉन्ट आकार";
+"Auto ( Recommended )" = "ऑटो (अनुशंसित)";
+"Customized" = "स्वनिर्धारित";
+"If custom font size is used, the actual font size will depend on the base font size and css style." = "यदि कस्टम फ़ॉन्ट आकार का उपयोग किया जाता है, तो वास्तविक फ़ॉन्ट आकार बेस फ़ॉन्ट आकार और सीएसएस शैली पर निर्भर करेगा।";
+"Left Panel" = "बाएं पैनल";
+"Show Outline Panel by Default" = "डिफ़ॉल्ट रूप से आउटलाइन पैनल दिखाएं";
+"Collapsible Outline on Left Panel" = "बाएं पैनल पर संकुचित रूपरेखा";
+"Save without asking when\nswitch files on side panel" = "बिना पूछे बिना सहेजें जब साइड पैनल पर \nswitch फ़ाइलें";
+"Edited files will be saved without asking when switch files in left panel. \nWhen \"Ask to keep changes when closing documents\"बाएं पैनल में फ़ाइलों को स्विच करते समय संपादित फ़ाइलों को सहेजा जाएगा। \nWhen \" = "Edited files will be saved without asking when switch files in left panel. \nWhen \"Ask to keep changes when closing documents\" is disabled in System Preference, this is always enabled.";
```
Comment (abnerlee) [inline hi_IN.lproj/Panel.strings:30]: This line is also not correct translated

File: hi_IN.lproj/Front.strings (hunk 3 above)
Comment (plutonium-239) [reply hi_IN.lproj/Front.strings:16]: Yes

File: hi_IN.lproj/Front.strings (hunk 4 above)
Comment (plutonium-239) [reply hi_IN.lproj/Front.strings:161]: just remove the extra symbols, i.e. `में खुलेगा` (would be 'more correct')

File: hi_IN.lproj/Welcome.strings (hunk 2 above)
Comment (plutonium-239) [reply hi_IN.lproj/Welcome.strings:84]: Sorry, looks like I jumbled it up.
should be
```
"Your license has exceeded the max devices numbers.\nIf you click \"Continue Activation\", this device will be activated and the oldest device will be unregistered automatically." = "आपका लाइसेंस अधिकतम डिवाइस संख्याओं को पार कर गया है। अगर आप \"Continue Activation\" क्लिक करते हैं, तो यह उपकरण सक्रिय हो जाएगा और सबसे पुराना उपकरण स्वचालित रूप से अपंजीकृत हो जाएगा" 
```

File: hi_IN.lproj/Panel.strings (hunk 5 above)
Comment (plutonium-239) [reply hi_IN.lproj/Panel.strings:30]: Should be
```
"Edited files will be saved without asking when switch files in left panel. \nWhen \"Ask to keep changes when closing documents\" is disabled in System Preference, this is always enabled." = "बाएं पैनल में फ़ाइलों को स्विच करते समय संपादित फ़ाइलों को सहेजा जाएगा। \nजब सिस्टम वरीयता में \"Ask to keep changes when closing documents\" अक्षम होता है, तो यह हमेशा सक्षम होता है।"
```

File: hi_IN.lproj/Panel.strings (hunk 1 above)
Comment (plutonium-239) [reply hi_IN.lproj/Panel.strings:240]: should be
```
"This option may be overridden by \"Close windows when quitting an application\" in System Preferences" = "इस विकल्प को ओवरराइड किया जा सकता है जब सिस्टम वरीयता में \"Close windows when quitting an application\" अक्षम हो।";
```

**Pull request merged**